glide.lock

```diff
@@ -38,11 +46,12 @@ imports:
   subpackages:
   - googleapis/rpc/status
 - name: google.golang.org/grpc
-  version: b8669c35455183da6d5c474ea6e72fbf55183274
+  version: f92cdcd7dcdc69e81b2d7b338479a19a8723cfa3
```

Comment:
Can we change it back to b8669c35455183da6d5c474ea6e72fbf55183274 to match snap version https://github.com/intelsdi-x/snap/blob/master/glide.lock